spec/features/user_spec.rb

```diff
@@ -399,6 +399,16 @@
         )
         click_button("Sign in")
       end
+
+      it "the admin user is redirected to the organisations list page on successful sign in" do
+        visit("/account/sign-in")
+        fill_in("user[email]", with: support_user.email)
+        fill_in("user[password]", with: support_user.password)
+        click_button("Sign in")
+        fill_in("code", with: otp)
+        click_button("Submit")
+        expect(page).to have_current_path("/organisations")
+      end
```

Comment:
Given that this is one single spec and the main change behind this is in the `two_factor_authentication_controller` I felt less inclined to convert this into a request spec